asyncapi/asb/Package.md

```diff
@@ -0,0 +1,23 @@
+Listen to [Microsoft Azure Service Bus](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true) from Ballerina.
+
+## Package overview
+
+The `ballerinax/trigger.asb` is a [Ballerina](https://ballerina.io/) trigger for Microsoft Azure Service Bus.
+This package provides the capability to access Microsoft Azure Service Bus messages from Service Bus queues, topics, and subscriptions, etc.
+
+### Compatibility
+
+|                            | Version               |
+|----------------------------|-----------------------|
+| Ballerina Language         | Swan Lake Beta3       |
```

Comment:
We can put Beta 4 since it's used to build & test?

Reply:
This is resolved now & changes are amended. Thank you.